src/home/WalletHome.test.tsx

```diff
@@ -293,6 +293,19 @@ describe('WalletHome', () => {
     expect(queryByTestId('cashInBtn')).toBeFalsy()
   })
 
+  it('shows beta tag when feature gate set to true', async () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
nit: reset this mock in a before each as these could have side effects on other tests